apps/qubit/modules/informationobject/actions/updatePublicationStatusAction.class.php

```diff
@@ -35,19 +35,60 @@ public function execute($request)
             if ($this->form->isValid()) {
                 // Update resource synchronously
                 $publicationStatusId = $this->form->getValue('publicationStatus');
-                $this->resource->setPublicationStatus($publicationStatusId);
-                $this->resource->save();
 
-                // Update descendants using job scheduler
+                $sql = 'UPDATE status
+                    JOIN information_object io ON status.object_id = io.id
+                    SET status.status_id = :publicationStatus
+                    WHERE status.type_id = :publicationStatusType
+                    AND io.id = :id';
+
+                $params = [
+                    ':publicationStatus' => $publicationStatusId,
+                    ':publicationStatusType' => QubitTerm::STATUS_TYPE_PUBLICATION_ID,
+                    ':id' => $this->resource->id,
+                ];
+
+                $descriptionsUpdated = QubitPdo::modify(
+                    $sql,
+                    $params
+                );
+
+                $response = QubitSearch::getInstance()->updateByQuery(
+                    [
+                        '_id' => $this->resource->id,
+                    ],
```

Comment:
Nitpick: no need to split this array into multiple lines.

Reply:
This has been removed completely so no longer an issue.